genesis-applier/genesis_applier.go

```diff
@@ -0,0 +1,36 @@
+package genesis_applier
+
+import (
+	"encoding/json"
+	"fmt"
+	"math/big"
+
+	"github.com/ethereum/go-ethereum/core"
+)
+
+type GenesisApplier struct {
```

Comment:
are we planning to add other functions to struct?

Think it might be cleaner if we make it pure function

```
func UpdateGenesisWithConfig(json []byte, config Config) ([]byte, error) {
// config can later include other things (including chainId that we want to override)
   ...
}

```

Reply:
I agree with the above unless we're going to add other functions

Reply:
good suggestion! refactored to be a pure function